models/repo_language_stats.go

```diff
@@ -20,11 +20,28 @@ type LanguageStat struct {
 	CommitID    string
 	IsPrimary   bool
 	Language    string             `xorm:"VARCHAR(30) UNIQUE(s) INDEX NOT NULL"`
-	Percentage  float32            `xorm:"NUMERIC(5,2) NOT NULL DEFAULT 0"`
+	Percentage  float32            `xorm:"-"`
+	Size        int64              `xorm:"NOT NULL DEFAULT 0"`
```

Comment:
`size` should have `index` tag on `xorm` if you want to `order by` with it.

Reply:
Results will be already filtered to only one repo so there will be less 10 mostly so that should not be problem